### PR TITLE
New/GUM 70 pull to refresh

### PR DESCRIPTION
**Pull request by R-Mielamud**

### CODE REVIEW

File: src/screens/mentee-home/mentee-home.tsx [hunk 1]
```diff
@@ -42,11 +42,19 @@ const MenteeHome: React.FC = () => {
...
   );
 
+  const reloadOpportunities = useCallback(() => {
+    dispatch(opportunityActions.loadOpportunities());
+  }, [dispatch]);
+
+  const reloadList = useCallback(() => {
+    reloadOpportunities();
+  }, [reloadOpportunities]);
+
```
Comment (katyakostikova) [inline src/screens/mentee-home/mentee-home.tsx]: Maybe it's enough to have only reloadOpportunities function?

**Review by katyakostikova — APPROVED** [review]

File: src/screens/mentee-home/mentee-home.tsx (hunk 1 above)
Comment (R-Mielamud) [reply src/screens/mentee-home/mentee-home.tsx]: There also will be a reloadNotifications function

File: src/screens/mentee-home/mentee-home.tsx [hunk 2]
```diff
@@ -42,11 +42,19 @@ const MenteeHome: React.FC = () => {
     []
   );
 
+  const reloadOpportunities = useCallback(() => {
```
Comment (liza-veis) [inline src/screens/mentee-home/mentee-home.tsx]: I think It's better to rename all such functions via load
```suggestion
  const loadOpportunities = useCallback(() => {
```

File: src/screens/opportunities/opportunities.tsx [hunk 3]
```diff
@@ -1,28 +1,29 @@
...
-  const styles = useStyles();
-
   const navigation = useAppNavigation();
   const dispatch = useAppDispatch();
 
   const { opportunitiesLoading, opportunities } = useAppSelector(
     state => state.opportunity
   );
 
+  const reload = useCallback(() => {
```
Comment (liza-veis) [inline src/screens/opportunities/opportunities.tsx]:
```suggestion
  const loadOpportunities = useCallback(() => {
```

**Review by liza-veis — APPROVED** [review]

**Review by jol79 — APPROVED** [review]

File: src/screens/mentee-home/mentee-home.tsx [hunk 4]
```diff
@@ -46,11 +46,19 @@ const MenteeHome: React.FC = () => {
     []
   );
 
+  const loadOpportunities = useCallback(() => {
+    dispatch(opportunityActions.loadOpportunities());
+  }, [dispatch]);
+
+  const loadList = useCallback(() => {
+    loadOpportunities();
+  }, [loadOpportunities]);
```
Comment (AleksandrTermenzhy) [inline src/screens/mentee-home/mentee-home.tsx:55]: Why do we need this?

Comment (R-Mielamud) [reply src/screens/mentee-home/mentee-home.tsx:55]: Знімок екрана 2022-04-01 о 16 37 59

**Review by yevhenprybytok — APPROVED** [review]

**Pull request merged**